dist/spec/payid-easy-checkout.txt

```diff
@@ -0,0 +1,728 @@
+
+   o  Recipient: Individual or entity receiving a payment (e.g.,
+      e-commerce merchant, charity).
+
+   o  Payer: Individual or entity originating a payment to a
+      "recipient".
+
+   o  Wallet: A device or application that holds funds (may be a non-
+      custodial wallet).
+
+   o  PayID Easy Checkout URL: The URL that is the result of the PayID
+      Easy Checkout protocol; can be used to redirect a client to a
+      wallet corresponding to a particular PayID.
+
+   The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+   "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and
+   "OPTIONAL" in this document are to be interpreted as described in
+   [RFC2119] and [RFC9174][].
+
+2.  Introduction
+
+   The PayID Easy Checkout Protocol is a minimal protocol that allows a
+   recipient (e.g., an online merchant or a charity) to request a
+   payment from a payer using only the payer's PayID.  Implementations
+   of the protocol should require little to no server-side engineering
+   efforts, while creating an improved and uniform user experience for
+   payers.
+
+   The main focus of the Protocol is on PayID Easy Checkout Discovery,
+   which defines how a PayID client can use a PayID to retrieve a PayID
+   Easy Checkout URL which represents a resource that the payer's
+   digital wallet can use to initiate a payment to the merchant.
+
+   Though the Section 8 of this specification provides an example usage
+   of a PayID Easy Checkout URL using Web Redirects, supplemental RFCs
+   are needed to define the different ways in which a PayID client can
+   utilize a PayID Easy Checkout URL.
+
+3.  PayID Easy Checkout Protocol
+
+   The PayID Easy Checkout Protocol can be used to facilitate an end-to-
+   end checkout flow between a payment recipient, such as an online
+   merchant, and a sending client, such as a wallet.
+
+   The protocol is comprised of two parts:
+
+
+
+
+Kramer, et al.          Expires January 24, 2021                [Page 3]
+
+Internet-Draft   Draft 1 - PayID Easy Checkout Protocol        July 2020
+
+
+   1.  PayID Easy Checkout Discovery
+
+   2.  PayID Easy Checkout URL Assembly
+
+3.1.  PayID Easy Checkout Discovery
+
+   PayID Easy Checkout Discovery extends [PAYID-DISCOVERY] by defining a
+   new link in the PayID metadata JRD returned by a PayID Discovery
+   query.  This link, defined in the JRD section (Section 4) of this
+   specification, includes the PayID Easy Checkout URL representing a
+   resource on the wallet which can be used to complete a payment.
+
+   Payment receivers who wish to initiate an Easy Checkout flow MUST
+   query the sender's PayID Discovery server to obtain a PayID Easy
+   Checkout URL.  PayID servers that wish to enable PayID Easy Checkout
+   MUST include a JRD Link conforming to the definition in the JRD
+   section (Section 4) of this paper in all PayID Easy Checkout
+   Discovery responses.
```

Comment:
There is a confusion around terminology here. We never define 
"Payment receiver" : Should it be "recipient"?
"Sender's PayID Discovery Server": Is this taken from PayID Discovery RFC?
"PayID server" is never defined here. Again since it is inverse role here for the PayID server (because it is on the sender's side), we must define it in this RFC and be explicit. I think a figure will really help.

Reply:
>"Payment receiver" : Should it be "recipient"?

yes

> "Sender's PayID Discovery Server": Is this taken from PayID Discovery RFC?

I got rid of "PayID Server" from the terminology and replaced it with "PayID Discovery Server", which is defined in the Terminology section

> "PayID server" is never defined here. Again since it is inverse role here for the PayID server (because it is on the sender's side), we must define it in this RFC and be explicit. I think a figure will really help.

See above